test/integration/index.js

```diff
@@ -7,7 +7,7 @@ const isIE = browserName === 'internet explorer'
 // const isIE9 = isIE && version === '9'
 // const isIE10 = isIE && version === '10'
 // const isIE11 = isIE && version === '11.103'
-const liveRegionWaitTimeMillis = 2000
+const liveRegionWaitTimeMillis = 10000
```

Comment:
Is this change still needed if we're only testing in Chrome?

Reply:
Yes unfortunately :(

Reply:
It might be worth updating the commit message, as at the minute it reads as though we're 'just' doing option 3, when it appears we're really doing _both_ 2 and 3?

Reply:
Yes, see point above 
> 3. Limit the browsers the tests run in

Reply:
Yes good point @36degrees, I'll update the commit message once the tests have passed once